Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1731,7 +1732,13 @@ private string GetAudioArguments(StreamState state)
 
             var channels = state.OutputAudioChannels;
 
-            if (channels.HasValue)
+            if (channels.HasValue
+                && (channels.Value != 2
+                    || (channels.Value == 2
```

Comment:
No need for this second check on `channels.value`

Reply:
Removed it.